src/bin/vip-import-validate-files.js

```diff
@@ -1,26 +1,31 @@
 #!/usr/bin/env node
 
+/**
+ * External dependencies
+ */
 import chalk from 'chalk';
-import fs from 'fs';
-import path from 'path';
 import url from 'url';
 
+/**
+ * Internal dependencies
+ */
 import command from '../lib/cli/command';
+import { getMediaImportConfig } from '../lib/media-import/config';
 import { trackEvent } from '../lib/tracker';
 import {
-	acceptedExtensions,
 	findNestedDirectories,
 	folderStructureValidation,
-	isFileSanitized,
-	doesImageHaveExistingSource,
-	logErrorsForIntermediateImages,
-	logErrorsForInvalidFileTypes,
-	logErrorsForInvalidFilenames,
 	summaryLogs,
+	validateFiles,
+	logErrors,
+	ValidateFilesErrors,
 } from '../lib/vip-import-validate-files';
 
 command( { requiredArgs: 1, format: true } )
-	.example( 'vip import validate-files <file>', 'Run the import validation against the file' )
+	.example(
+		'vip import validate-files <folder_name>',
+		'Run the import validation against the folder of media files'
```

Comment:
Nit & Non-blocker & just a question: I am not sure where else we prompt the word "folder" but do you think that "directory" is better term?

Reply:
> Nit & Non-blocker & just a question: I am not sure where else we prompt the word "folder" but do you think that "directory" is better term?
Yes, it could be a better term in the command line context. I used "folder" to keep it consistent with the implementation variables.

Reply:
it's also a term being used in the current version.